app/src/test/java/org/oppia/android/app/accessibility/FakeAccessibilityServiceTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.android.app.accessibility
+
+import android.view.View
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.oppia.android.util.accessibility.FakeAccessibilityService
+
+/** Tests for [FakeAccessibilityService]. */
+class FakeAccessibilityServiceTest {
+  private var accessibilityManager = FakeAccessibilityService()
+
+  @Test
+  fun testInitialState() {
```

Comment:
Prefer splitting this up such that we're verifying the initial state of both properties.

Reply:
Done.